mobile/src/components/auth/components/sign-in-form/sign-in-form.tsx

```diff
@@ -1,24 +1,53 @@
 import React, { FC } from 'react';
 
-import { RootScreenName } from '~/common/enums/enums';
-import { Button, Link, Text, View } from '~/components/common/common';
+import { UserSignInRequestDto } from '~/common/types/types';
+import { Button, Input, Text, View } from '~/components/common/common';
+import { useAppForm } from '~/hooks/hooks';
+import { userSignIn as userSignInValidationSchema } from '~/validation-schemas/validation-schemas';
+
+import { DEFAULT_SIGN_IN_PAYLOAD } from './common/constants';
+import { styles } from './styles';
 
 type Props = {
   onSubmit: () => void;
 };
 
 const SignInForm: FC<Props> = () => {
+  const { control, errors } = useAppForm<UserSignInRequestDto>({
+    defaultValues: DEFAULT_SIGN_IN_PAYLOAD,
+    validationSchema: userSignInValidationSchema,
+  });
+
   return (
     <>
-      <Text>Sign In</Text>
+      <Text style={styles.title}>Sign in</Text>
       <View>
-        <Button
-          label="Sign in"
-          onPress={(): void => {
-            // TODO: handle press
-          }}
-        />
-        <Link label="Go to Sign Up" to={{ screen: RootScreenName.SIGN_UP }} />
+        <View style={styles.inputWrapper}>
+          <Input
+            label="Email"
+            placeholder="Enter your email"
+            name="email"
+            control={control}
+            errors={errors}
+          />
+        </View>
+        <View style={styles.inputWrapper}>
+          <Input
+            label="Password"
+            placeholder="Enter your password"
+            name="password"
+            control={control}
+            errors={errors}
+          />
+        </View>
+        <View style={styles.buttonWrapper}>
+          <Button
+            label="Sign in"
+            onPress={(): void => {
+              // TODO: handle press
+            }}
```

Comment:
Connect it to server, you should add sign-in action, and save user to the store, and then add condition in the root-navigation - if there user in the store, then render app navigation, otherwise auth navigation. Also set sign-in as a first screen

Reply:
In progress) TY